backend/src/admin/admin.module.ts

```diff
@@ -4,7 +4,7 @@ import { AdminController } from './admin.controller';
 import { AdminService } from './admin.service';
 import { Admin } from './admin.entity';
 import { AuthenticationModule } from '../authentication/authentication.module';
-import { Junior } from '../junior/junior.entity';
+import { Junior } from '../junior/entities/index';
```

Comment:
`import { Junior } from '../junior/entities/index';` can be changed to `import { Junior } from '../junior/entities';` as it will be imported as a module. In my opinion it looks a bit nicer. :)